var/spack/repos/builtin/packages/clingo-bootstrap/package.py

```diff
@@ -49,15 +69,45 @@ class ClingoBootstrap(Clingo):
     def cmake_py_shared(self):
         return self.define("CLINGO_BUILD_PY_SHARED", "OFF")
 
-    def cmake_args(self):
-        args = super(ClingoBootstrap, self).cmake_args()
-        args.extend(
-            [
-                # Avoid building the clingo executable
-                self.define("CLINGO_BUILD_APPS", "OFF"),
-            ]
+    @when("+optimized")
```

Comment:
This seems to have no effect?

Reply:
Didn't you fix this @becker33 ?

Reply:
Try:
```
@run_before("cmake", when="+optimized")
```
(the change was part of multiple build-system.